core/files/css/global-css.php

```diff
@@ -1,160 +0,0 @@
-<?php
```

Comment:
@louiswol94 The removal of this file is styling causing the test to fail: https://github.com/elementor/elementor/actions/runs/11274157400/job/31352932705?pr=28938.

Could you please have a look at this?